source/newcommitter.md

```diff
@@ -75,17 +75,19 @@ later in this document.
 <a name="NewCommitter-Summary"></a>
 ### Summary
 
-1. Call a vote (templates/committerVote.txt)
-1. Close the vote
+1. Discuss the proposed committer/PMC member. If positive, call a vote (templates/committerVote.txt)
+1. Close the vote (templates/closeVote.txt)
 1. If the result is positive, invite the new committer (templates/committerInvite.txt)
 
 If they accept, then:
 
-1. Accept the committer (templates/committerAccept.txt)
-1. Wait until we see that receipt of CLA is recorded
-1. Request creation of the committer account (template/committerCreate.txt)
+1. If they already have an Apache id, grant appropriate commit privileges.
+1. If they have already filed an ICLA, request creation of the committer account.
```

Comment:
And here would be nice to have a link to the list of ICLAs already filed.

Reply:
Not really, the user would have to tell us that they have filed a CLA. Only officers and members can see them and they have to provide the email address or name it was filed under.